server/src/routes/api/v2/wallets.js

```diff
@@ -67,7 +67,8 @@ router.put('/token/:walletAddress', auth.required, async (req, res) => {
  */
 router.get('/', auth.required, async (req, res, next) => {
   const { phoneNumber, accountAddress } = req.user
-  const userWallet = await UserWallet.findOne({ phoneNumber, accountAddress }, { contacts: 0, firebaseToken: 0 })
+  const network = req.query.network || 'fuse'
+  const userWallet = await UserWallet.findOne({ phoneNumber, accountAddress, networks: network }, { contacts: 0, firebaseToken: 0 })
```

Comment:
if `networks` is array you're comparing an array and string (['fuse'] == 'fuse'). 
This works in mongoose?

Reply:
It works, I tried it both in the compass itself as well as run the code locally.
Surprised me too...